__tests__/app.test.js

```diff
@@ -216,3 +229,98 @@ describe.only('POST/api/reviews/:review_id/comments', () => {
+            expect(body).toHaveProperty('category', expect.any(String))
+            expect(body).toHaveProperty('designer', expect.any(String))
+            expect(body).toHaveProperty('owner', expect.any(String))
+            expect(body).toHaveProperty('review_body', expect.any(String))
+            expect(body).toHaveProperty('review_img_url', expect.any(String))
+            expect(body).toHaveProperty('created_at', expect.any(String))
+            expect(body).toHaveProperty('votes', 11)
+        })
+    });
+    test('can take a a vote down that will take votes below 0', () => {
+        return request(app).patch('/api/reviews/1').send(voteDown)
+        .then(({body}) => {
+            expect(body).toHaveProperty('votes', -9)
+        })
+    });
+    //error handling
+    test('if id valid but not found, returns 404 ID not found (valid body sent', () => {
+        return request(app).patch('/api/reviews/14').send(voteUp)
+        .expect(404).then(({text}) => {
+            expect(text).toBe("Review Id not found")
+        })
+    });
+    test('if id not not valid, returns 400 not valid ID (valid body sent)', () => {
+        return request(app).patch('/api/reviews/cheese').send(voteUp)
+        .expect(400).then(({body}) => {
+            expect(body.msg).toBe('Not valid Review Id')
+        });
+    });
+    test('if sent body does not contain inc_votes property, fails with 400 bad request', () => {
+        return request(app).patch('/api/reviews/1').send({i_votes: 100})
+        .expect(400).then(({body}) => {
+            expect(body.msg).toBe("Bad Request - Invalid info sent")
+        });
+    });
+    test('if inc_votes contains an invalid value, fails with 400 bad request ', () => {
+        return request(app).patch('/api/reviews/1').send({i_votes: 'abc'})
+        .expect(400).then(({body}) => {
+            expect(body.msg).toBe("Bad Request - Invalid info sent")
+        });
+    });
+    test('can ignore extra properties and complete patch aslong as valid info is available', () => {
+    const voteUpWithExtraProperties = {inc_votes: 10, extraproperty: true}
+        return request(app).patch('/api/reviews/1').send(voteUpWithExtraProperties)
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toHaveProperty('review_id', 1)
+            expect(body).toHaveProperty('title', expect.any(String))
+            expect(body).toHaveProperty('category', expect.any(String))
+            expect(body).toHaveProperty('designer', expect.any(String))
+            expect(body).toHaveProperty('owner', expect.any(String))
+            expect(body).toHaveProperty('review_body', expect.any(String))
+            expect(body).toHaveProperty('review_img_url', expect.any(String))
+            expect(body).toHaveProperty('created_at', expect.any(String))
+            expect(body).toHaveProperty('votes', 11)
+        })
+    });
+});
+
+describe.only('GET/api/users', () => {
```

Comment:
There is a .only on here, normally I would reject any pull request with a .only on as we dont know that any of the other tests are passing and we need to make sure that everything else works whilst the new feature is implemented too